nomt/src/store.rs

```diff
@@ -73,6 +73,40 @@ impl Store {
         Ok(value)
     }
 
+    /// Given a set of page IDs, attempt to load each of them.
+    /// After the first `None`, all subsequent entries are `None`.
+    pub fn load_pages_dependent(&self, pages: Vec<PageId>) -> anyhow::Result<Vec<Option<Vec<u8>>>> {
```

Comment:
Not sure if it's clear what makes them "dependent" exactly worth mentioning that in the docs.